tests/plugins/toolsets/test_skills.py

```
@@ -296,3 +304,161 @@ def test_SkillsFetcher_one_liner_uses_title_for_remote_skills():
+    dns-debug, pod-oom".
+    """
+    good = tmp_path / "good"
+    mount = tmp_path / "mount"
+    _write_fs_skill(good, "dns-debug", "dns steps")
+    _write_fs_skill(mount, "pod-oom", "Check memory limits")
+    toolset = SkillsToolset(additional_search_paths=[str(good), str(mount)])
+    fetcher = toolset.tools[0]
+
+    shutil.rmtree(mount)  # the mount goes away for a cycle
+
+    skill, authoritative = fetcher._find_filesystem_skill("pod-oom")
```

Ruff (0.16.2)

[warning] Unpacked variable `skill` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)